packages/delegate/src/transforms/ExpandAbstractTypes.ts

```diff
@@ -187,24 +212,49 @@ function expandAbstractTypes(
                   }
                 });
               }
+            } else if (
+              selection.kind === Kind.FIELD &&
+              extendedInterface != null &&
+              extendedInterface.includes(selection.name.value)
```

Comment:
Could this be set up as a map of booleans or whatever to avoid includes in favor of map lookup which will have better performance.

Reply:
was thinking that same thing, and was going to ask about that as a point of style... +1